yarpcconfig/builder.go

```diff
@@ -125,6 +126,13 @@ func (b *builder) Build() (yarpc.Config, error) {
 				continue
 			}
 		}
+		if o := c.Stream; o != nil {
+			ob.Stream, err = buildStreamOutbound(o, transports[o.TransportSpec.Name], b.kit)
+			if err != nil {
+				errs = multierr.Append(errs, fmt.Errorf(`failed to configure stream outbound for %q: %v`, ccname, err))
```

Comment:
Backticks are extra for this, but I find it hard to care about that level of consistency.

Reply:
pedantry is good; use quotes